### DISCUSSION TITLE
"[Errno 2] No such file or directory" when uploading dataset

### DISCUSSION THREAD

**Discussion by melissawyleung** [asreview/asreview#1447] [Troubleshooting (report errors in the "Issues" tracker)] [answered]
Hi all,

When I try to upload a dataset in step 2 of creating a project, I get the error "[Errno 2] No such file or directory".
I have retried with the file stored in different locations, different supported file types, and a limited set of supported variables in the dataset, but with no success so far.

I have also asked for help from colleagues, who have successfully uploaded datasets.
As they did this from their own devices, I am wondering if the problem may be to do with my using a UU laptop to upload a dataset from.

Regardless of the cause, I would appreciate any help I can get!

**Comment by J535D165** [comment]
Sorry for inconvenience, can you show me the full output in the terminal?

**Comment by tsmeele** [comment]
Have you installed ASReview 1) locally or 2) are you using a ResearchCloud based environment?

In case 2) then ASReview might search for dataset files on the cloud workspace (the remote desktop) rather then on your laptop.

**Comment by melissawyleung** [comment, marked as answer]
Thank you for thinking with me!

The solution was to shorten the project name.

**Reply by J535D165** [reply]
This issue will be resolved in version 1.3 (coming soon).
